histomicstk/AreaOpenLabel.py

```diff
@@ -0,0 +1,58 @@
+import histomicstk as htk
+import numpy as np
+import scipy.ndimage.measurements as ms
+
+
+def AreaOpenLabel(Label, Area):
+    """Removes small objects from label image.
+
+    Parameters:
+    -----------
+    Label : array_like
+        A uint32 type label image generated by segmentation methods.
+    Area : int
+        Area threshold for objects. Objects with fewer than 'Area' pixels will
+        be zeroed to merge with background.
+
+    Notes:
+    ------
+    Objects are assumed to have positive nonzero values. Label image will be
+    condensed during processing.
+
+    Returns:
+    --------
+    Split : array_like
+        A uint32 label where objects with pixels < Area are removed.
+
+    See Also:
+    ---------
+    CondenseLabel, ShuffleLabel, SplitLabel, MaxwidthOpenLabel
+    """
+
+    # copy input image
+    Opened = Label.copy()
+
+    # condense label image
+    if np.unique(Opened).size-1 != Opened.max():
+        Opened = htk.CondenseLabel(Opened)
+
+    # count pixels in each object
+    Counts, Edges = np.histogram(Opened, bins=Opened.max()+1)
+
+    # get locations of objects in initial image
+    Locations = ms.find_objects(Opened)
+
+    # iterate through objects, zeroing where needed
+    for i in np.arange(1, Counts.size):
+        if Counts[i] < Area:
+
+            # extract object from label image
+            Template = Opened[Locations[i-1]]
+
+            # label mask of object 'i'
+            Template[Template == i] = 0
+
+    # condense to fill gaps
+    Opened = htk.CondenseLabel(Opened)
+
+    return Opened
```

Comment:
We can do this using [regionprops](http://scikit-image.org/docs/dev/api/skimage.measure.html#skimage.measure.regionprops) but this is probably faster. Wish they adapt `regionprops` in scikit-image to behave similar to the matlab one wherein we can request a selected subset of the properties to be computed.